src/components/WindowSideBarButtons.js

```diff
@@ -16,23 +17,148 @@ export class WindowSideBarButtons extends Component {
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
+    this.handleKeyUp = this.handleKeyUp.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleTabRef = this.handleTabRef.bind(this);
+
+    this.keys = {
+      end: 35,
+      home: 36,
+      up: 38,
+      down: 40,
```

Comment:
MUI uses `event.key` and compares that to things like `Home` and `ArrowUp`.  This may be more reliable that the key codes? https://github.com/mui-org/material-ui/blob/7e1a0f1e1574d36188d333d8a91288e6f516db09/packages/material-ui/src/MenuList/MenuList.js#L43-L82

Reply:
Yes, you're right, `keyCode` is marked as deprecated. But unfortunately 'key' doesn't work for Safari. I've switched to 'key' and 'which' (this combination should work in all modern browsers)